### ISSUE TITLE
rtorrent can't download completely

### ISSUE THREAD

**Issue by colinhd8** [rakshasa/rtorrent#529]
sometimes it's seems that the torrent downloaded completely, but show as downloading. And it will continue forever. Below is the information:
Done: 99.98%
seeds: 54/177
peers: 2/9

DL: 0
UL: 555.1KB/s

![chunks](https://cloud.githubusercontent.com/assets/5259758/21127620/528432b4-c12f-11e6-9775-55cc50cdac63.PNG)
there is a queued chunk, i think it's the problem.

**Comment by pyroscope** [comment]
Try to close it with ^K (and be prepared for a crash on closing). If it crashes, it's a known problem.

**Comment by colinhd8** [comment]
Yes, in the past time i had the same problem and i close it with ^k then it crashed.
I will do the same action for this torrent and confirm what's the result.
Thanks for reply and hope someone will fix it.
